modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOSTController.scala

```diff
@@ -121,16 +60,39 @@ final case class GHOSTController[F[_]: Sync](ghostClient: GHOSTClient[F],
 }
 
 object GHOSTController {
+  private def ifuConfig(ifuNum: IFUNum,
+                        ifuTargetType: IFUTargetType,
+                        coordinates: Coordinates,
+                        bundleConfig: BundleConfig): Configuration = {
+    def cfg[P: Show](paramName: String, paramVal: P) =
+      Configuration.single(s"${ifuNum.show}.$paramName", paramVal)
+      cfg("target", ifuTargetType) |+|
+      cfg("type", DemandType.DemandRADec.demandType) |+|
+      cfg("ra", coordinates.ra.toAngle.toDoubleDegrees) |+|
+      cfg("dec", coordinates.dec.toAngle.toDoubleDegrees) |+|
+      cfg("bundle", bundleConfig.show)
+  }
+
+  private def ifuPark(ifuNum: IFUNum): Configuration = {
+    def cfg[P: Show](paramName: String, paramVal: P) =
+      Configuration.single(s"${ifuNum.ifuStr}.$paramName", paramVal)
+      cfg("target", IFUTargetType.NoTarget.targetType) |+|
+      cfg("type", DemandType.DemandPark.demandType)
+  }
+
   sealed abstract class BundleConfig(val configName: String) {
     def determineType(t: IFUTargetType): BundleConfig = t match {
       case IFUTargetType.SkyPosition => BundleConfig.Sky
       case _                         => this
     }
   }
+
   object BundleConfig {
     case object Standard extends BundleConfig(configName = "IFU_LORES")
     case object HighRes  extends BundleConfig(configName = "IFU_HRES")
     case object Sky      extends BundleConfig(configName = "IFU_SKY")
+
+    implicit val showBundleConfig: Show[BundleConfig] = Show.show(_.configName)
```

Comment:
I'd like use not to use `Show` for this and define a `GiapiParam` typeclass instead. That probably belongs to another PR though

Reply:
Yes, we have a task specifically for this, I think, to do the same for GPI. I was going to look at that one next.